rs/pocket_ic_server/src/main.rs

```diff
@@ -181,6 +181,7 @@ async fn start(runtime: Arc<Runtime>) {
         .nest("/instances", instances_routes::<AppState>())
         // All HTTP gateway routes.
         .nest("/http_gateway", http_gateway_routes::<AppState>())
+        .fallback(|| async { (StatusCode::NOT_FOUND, "") })
```

Comment:
This fallback is triggered for all paths under /http_gateway that are not in http_gateway_routes? 
If so, how does the NOT_FOUND differ from the normal return value?

Reply:
> This fallback is triggered for all paths under /http_gateway that are not in http_gateway_routes?

This fallback refers to the root of the URL path, not to `/http_gateway` specifically.

> If so, how does the NOT_FOUND differ from the normal return value?

You're right that this is already the default at the moment, but specifying it explicitly makes the behavior predictable across axum version bumps.